examples/core/shared/src/main/scala/zio/logging/example/SimpleApp.scala

```diff
@@ -15,18 +15,20 @@
  */
 package zio.logging.example
 
+import zio._
+import zio.internal.Platform
 import zio.logging.{ ConsoleLoggerConfig, consoleLogger }
-import zio.{ ExitCode, Runtime, Scope, ZIO, ZIOAppArgs, ZIOAppDefault, ZLayer }
 
 object SimpleApp extends ZIOAppDefault {
 
   override val bootstrap: ZLayer[ZIOAppArgs, Any, Any] =
-    Runtime.removeDefaultLoggers >>> consoleLogger(ConsoleLoggerConfig.default)
+    if (Platform.isJS) ZLayer.empty
+    else Runtime.removeDefaultLoggers >>> consoleLogger(ConsoleLoggerConfig.default)
```

Comment:
on JS this seems to remove all logging... ideas?

Reply:
personally i do not have experience with scala JS, but if there are differences i think is better to do example specific for jvm and JS

Reply:
anyway, with that condition, it look like that for platform JS it will not use `consoleLogger(ConsoleLoggerConfig.default)`

Reply:
fixed it, all platforms use the same setup again. And discovered it wasn't working before because the scala-java-time-tzdb dependency was missing. Added them to 'core' because it is always required when using zio-logging.